scripts/postinstall.js

```diff
@@ -21,23 +21,26 @@ if (fs.existsSync(targetFilePath)) {
 
 /* OUI -> EUI Aliases */
 const { INIT_CWD, PWD = process.cwd() } = process.env;
-// Only run when installed as a dep
+
+// Only run when installed as a dep, duplicated `oui` theme files as `eui`
 if (!INIT_CWD?.startsWith?.(PWD)) {
-  const shell = require('shelljs');
-// Clean up and recreate the folders
-  shell.rm('-rf', 'src/themes/eui');
-  shell.rm('-rf', 'src/themes/eui-amsterdam');
-  shell.mkdir('-p', 'src/themes/eui');
-  shell.mkdir('-p', 'src/themes/eui-amsterdam');
-
-  shell.cp('-fR', 'src/themes/oui/oui_colors_dark.scss', 'src/themes/eui/eui_colors_dark.scss');
-  shell.cp('-fR', 'src/themes/oui/oui_colors_light.scss', 'src/themes/eui/eui_colors_light.scss');
-  shell.cp('-fR', 'src/themes/oui/oui_globals.scss', 'src/themes/eui/eui_globals.scss');
-
-  shell.cp('-fR', 'src/themes/oui-cascadia/global_styling', 'src/themes/eui-amsterdam/global_styling');
-  shell.cp('-fR', 'src/themes/oui-cascadia/overrides', 'src/themes/eui-amsterdam/overrides');
-  shell.cp('-fR', 'src/themes/oui-cascadia/oui_cascadia_colors_dark.scss', 'src/themes/eui-amsterdam/eui_amsterdam_colors_dark.scss');
-  shell.cp('-fR', 'src/themes/oui-cascadia/oui_cascadia_colors_light.scss', 'src/themes/eui-amsterdam/eui_amsterdam_colors_light.scss');
-  shell.cp('-fR', 'src/themes/oui-cascadia/oui_cascadia_globals.scss', 'src/themes/eui-amsterdam/eui_amsterdam_globals.scss');
+  // Clean up and recreate the folders
+  fs.rmSync('src/themes/eui', { recursive: true, force: true });
+  fs.rmSync('src/themes/eui-amsterdam', { recursive: true, force: true });
+
+  const copyDirectory = (src, dest) => {
+    fs.mkdirSync(dest, { recursive: true });
+
+    for (const entry of fs.readdirSync(src, { withFileTypes: true })) {
+      const srcItem = path.join(src, entry.name);
+      const destItem = path.join(dest, entry.name);
+
+      if (entry.isDirectory()) copyDirectory(srcItem, destItem);
+      else fs.copyFileSync(srcItem, destItem);
```

Comment:
Again, why sync?

Reply:
No good reason. Just because existing code was sync.
BTW, these are just temporary while we alias.